app/public/src/pages/component/lobby-menu/sprite-debug.css

```diff
@@ -0,0 +1,51 @@
+.sprite-debug-root {
+    width: 100%;
+    height: 100%;
+
+    --sidebar-offset: 90px;
+}
+
+/* TODO Remove this after the absolute sidebar update goes in */
+.sprite-debug-root aside.sidebar {
```

Comment:
This update will be provided in `.main-sidebar.css` with [this PR](https://github.com/keldaanCommunity/pokemonAutoChess/pull/918)  but I kept this fallback in case we don't end up merging that one